ui/app/templates/vault/cluster/access/oidc.hbs

```diff
@@ -0,0 +1,59 @@
+{{#if this.header}}
+  <PageHeader as |p|>
+    <p.levelLeft>
+      <h1 class="title is-3">
+        OIDC Provider
+      </h1>
+    </p.levelLeft>
+  </PageHeader>
+  <div class="box is-fullwidth is-sideless is-flex-between is-box-shadowless is-marginless">
+    <p>
+      Configure Vault to act as an OIDC identity provider, and offer
+      {{"Vault’s"}}
+      various authentication
+      {{#if this.isCta}}
+        <br />
+      {{/if}}
+      methods and source of identity to any client applications.
+      <LearnLink @path="/tutorials/vault/oidc-auth">
```

Comment:
make sure this link is right. There is the oidc auth and oidc as a provider. I was confused about this and had to check with Yankun about the link. https://learn.hashicorp.com/tutorials/vault/oidc-identity-provider

Reply:
Good catch! I copied this over from the header component I removed and didn't pay attention to it. I'll update it to the proper link.